expo/tests/gbe/test_edit_combo.py

```diff
@@ -16,7 +16,7 @@ class TestCreateCombo(TestCase):
     def setUp(self):
         self.factory = RequestFactory()
         self.client = Client()
-        self.troupe_string = 'a one-off group of performers working together'
+        self.troupe_string = 'a one-off group of performers working     together'
```

Comment:
It's coming from trying to fix the pep8 errors in either gbe_text or gbe_forms_text.